sdk/nodejs/apigee/v1/organizationDeveloperAppKey.ts

```diff
@@ -1,173 +0,0 @@
-// *** WARNING: this file was generated by the Pulumi SDK Generator. ***
```

Comment:
Yes, I forgot to mention this. I think three resources are gone now because I wasn't able to find anything that looks like a `name` property there. I believe it generates a warning in the output. I'll make sure to open an issue to track those.

Reply:
https://github.com/pulumi/pulumi-google-native/issues/103